index.html

```diff
@@ -19,7 +19,7 @@
                 &lt;script src='stream.min.js'&gt;&lt;/script&gt;
             </pre>
 
-            <a class='download' href="stream.min.js">Download stream.js</a>
+            <a class='download' href="stream.js.zip">Download stream.js</a>
```

Comment:
Why zip it? A direct link to a minified version may be a better choice. That way people may even be able to hotlink directly to us, meaning they may be able to make use of some browser caching capacity.